pyscript.core/src/stdlib/pyscript/display.py

```diff
@@ -150,5 +150,12 @@ def display(*values, target=None, append=True):
         target = current_target()
 
     element = document.getElementById(target)
+
+    # if element is a <script type="py">, it has a 'target' attribute which
+    # points to the visual element holding the displayed values. In that case,
+    # use that.
+    if element.tagName == 'SCRIPT' and hasattr(element, 'target'):
```

Comment:
it's a lazy accessor and for scripts is present "*by contract*" ... I don't see any harm in double checking this but I wonder if you had any issue without that check because it's guaranteed to work as per *polyscript* documentation.

not a blocker, rather curiosity, thanks.

Reply:
no, I was just trying to prevent an exception in case you use `target='xxx'` where `#xxx` is a normal non-polyscript tag